External/SPEC/CINT2017rate/523.xalancbmk_r/CMakeLists.txt

```diff
@@ -8,6 +8,8 @@ speccpu2017_benchmark(RATE)
 
 set(CMAKE_CXX_STANDARD 14)
 
+add_compile_options(-fdelayed-template-parsing)
```

Comment:
Do we need to check if the flag is available to not break when building with compilers that don't have the flag (looks like it get added to Clang in 2011, so we should be good there, but llvm-test-suite also supports building with other compilers, which may not have the flag)

Reply:
Good idea, done